scripts/modules/elastic_stack.py

```diff
@@ -636,6 +636,8 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6"):
```

Comment:
maybe also the link to the documentation for the rest of users https://github.com/elastic/apm-integration-testing#logging-in